CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file.
 
+## \[6.4.4\]
+
+- Add `replace_trigger` to `_slurm_instance` for fine-grained control over
+  instance replacement
+- Add support for installing NVIDIA open drivers
+- Support specifying monitoring agent in Packer
+- TPU: update supported docker images list
+
```

Comment:
Please add 

- fix: prevent controller from mounting munge key by NFS
- fix: retry DNS lookups during early setup process
- fix: ensure VM always boots into kernel against which NVIDIA drivers were built
- add support for PMIx to image building solution (images published _after_ this release will contain PMIx)
- enable building Slurm on Debian 12 images (not yet officially supported or tested)

Reply:
An additional line to add
- fix: Removes unused parameter from `nodeset_switch_lines` function

Reply:
@mr0re1 please add the additional line requested by @alyssa-sm . I would add context like

```
fix: resolve bug in enable_configure by removing unused parameter from nodeset_switch_lines function
```